### PR TITLE
feat: add method buildTransactionWithReceiptPayload

### PR DESCRIPTION
**Pull request by nachomazzara**
- Add `buildTransactionWithReceiptPayload`
- Add circle

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    docker:
+      - image: circleci/node:9.11.1
+    steps:
+      - checkout
+      - run:
+          name: Install system dependencies
+          command: sudo apt-get update && sudo apt install -y libusb-1.0-0 libusb-1.0-0-dev
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
```
Comment (NicoSantangelo) [inline .circleci/config.yml]: I think it's better to use `package-lock.json` to avoid cache conflicts

File: src/modules/transaction/types.ts [hunk 2]
```diff
@@ -27,3 +30,7 @@ export enum NetworkName {
   kovan = 'kovan',
   rinkeby = 'rinkeby'
 }
+
+interface Receipt {
+  logs: any[]
```
Comment (NicoSantangelo) [inline src/modules/transaction/types.ts]: Is it possible to type this? or is it way too dynamic?

Comment (nachomazzara) [reply src/modules/transaction/types.ts]: It has events logs on it, `args` of events could be dynamic. 

I can do something like
```ts
logs: {
name: string,
args: any,
[key: string]: any
}
```

Is it worth?

Comment (NicoSantangelo) [reply src/modules/transaction/types.ts]: think so! it allows VSCode to autocomplete .name and .args at least

**Comment by NicoSantangelo** [comment]
🎉

**Pull request merged**